src/test/ui/param-bounds-ignored.rs

```diff
@@ -9,12 +9,14 @@
 // except according to those terms.
 
 // must-compile-successfully
+#![allow(dead_code, non_camel_case_types)]
```

Comment:
Could you `warn(ignored_generic_bounds)` and add `//~ WARN ...` annotations to the file?
It's very hard to see what is tested, what should be reported, and what shouldn't be reported based on `stderr` file alone.

Reply:
Ah, I didn't even know `//~ WARN` was a thing in ui tests. How do the two interact?

Reply:
Done.